releasenotes/notes/31095.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: networking
+issue:
+  - 31095
+releaseNotes:
+  - |
+    **Fixed** an issue where app call itself by podIP in TPROXY mode would be intercepted only by inbound.
```

Comment:
Can you make it clear? Adding `neither intercepted by inbound nor outbound`

Reply:
sure